### ISSUE TITLE
Accent in value of one of the values of payer_rbits fails the payment

### ISSUE THREAD

**Issue by thomas-oo** [wepay/NodeJS-SDK#20]
When you do wepay.call('/checkout/create', {......., payer_rbits:rbits] where rbits is an array with one object being an address rbit, if the address's city includes an accent, it fails.

Sample rbit:
```
              receive_time: credit_card.created_at.getTime(),
              source: 'user',
              type: 'address',
              properties: {
                address: {
                  address1: credit_card.address.street,
                  city: credit_card.address.city,
                  region: credit_card.address.province,
                  postal_code: credit_card.address.postal_code,
                  country: credit_card.address.country
                }
              }
```

In our case, city was Montréal, failing the call. Making the same call with Montreal was accepted.

Stacktrace: 
```
Error: Input could not be parsed
at ....
at IncomingMessage.<anonymous> (/app/node_modules/wepay/lib/wepay.js:84:25)
```

I have not tested if this issue where you include an accented character in any other places fails the call but I would assume it might.

**Comment by Jalepeno112** [comment]
Just created a pull request that should fix the issue.  The SDK is calculating and setting the Content-Length header incorrectly.  When the request hits the API, this mismatch causes the request to fail.  

Removing the calculation from the SDK and leaving it to the underlying `https` module fixed the issue for me.  The SDK could also improve the way it calculates Content-Length, but I'll leave it to the owners of the SDK to make the final decision.

**Comment by camerona93** [comment]
@thomas-oo I've merged @Jalepeno112's pull request and released version 0.0.9 to npm, so it should be available for you now.

Please reopen the issue if you have further problems. Thanks!
